src/main/target/HAKRCF405/target.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * This file is part of Cleanflight.
```

Comment:
This is an old version of the file header. Please replace with a current version, like all existing files have.

Reply:
OK ，I will modify